ViAn/GUI/mainwindow.cpp

```diff
@@ -170,11 +157,35 @@ void MainWindow::on_play_pause_button_clicked() {
  * Calls upon video player function which in turn increases the playback speed
  */
 void MainWindow::on_increase_speed_button_clicked(){
+    ui->decrease_speed_button->setEnabled(true);
     mvideo_player->inc_playback_speed();
     double curr_speed_factor = 1/mvideo_player->get_speed_multiplier();
     std::ostringstream speed_str;
     speed_str << curr_speed_factor;
+    QString speed = QString::fromStdString(speed_str.str());
```

Comment:
You can convert a double to QString easier like this `QString::number(valueAsDouble);`